ucp/_libs/ucp_py.pyx

```diff
@@ -181,55 +181,55 @@ class ListenerFuture(concurrent.futures.Future):
     def __del__(self):
         pass
 
-cdef class ucp_py_ep:
+cdef class EndPoint:
```

Comment:
This is definitely a nitpick - Is it `EndPoint` or should it be `Endpoint`? AFAIK _endpoint_ is a single word.

Reply:
good point :)